### PR TITLE
Merge method fixes

### PR DESCRIPTION
**Pull request by KevinGrandon**

### CODE REVIEW

File: binary [hunk 1]
```diff
@@ -30,6 +30,7 @@ program.on('error', function (err) {
 program.on('exit', function(code){
   // Wait few ms for error to be printed.
   setTimeout(function(){
+		console.log("Do we get in here?");
```
Comment (KevinGrandon) [inline binary]: Remove this console, then let's get this PR merged in?

Comment (akleiner2) [reply binary]: Yeah, whoops.

Comment (akleiner2) [reply binary]: Done

**Comment by KevinGrandon** [comment]
Let's get this in and work in smaller commits! We're almost there.

**Pull request merged**